File: src/routes/kpiYear.js
const routering = require('express').Router();
const { validateToken } = require("../jwt/jeisonWebToken");
const yearKpi = require('../controllers/yearKpi')

routering.get('/year',validateToken,(req,res)=>{
    console.log("Entro por ejecuciones para el año actual")
    yearKpi.kpiYear(req,res);
})

module.exports = routering

File: src/routes/kpiYear.py
from flask import Blueprint, request, jsonify
from controllers.yearKpi import kpiYear

routerYearKpi = Blueprint('routering', __name__)

@routerYearKpi.route('/year', methods=['GET'])
def year_kpi_route():
    print("Entro por ejecuciones para el año actual")
    return kpiYear(request, jsonify)